Documentation/guides/k8s-aws-ansible/install.rst

```diff
@@ -0,0 +1,1430 @@
+    aws ec2 create-tags --resources $M2 --tags Key=role,Value=master
+    aws ec2 modify-instance-attribute --instance-id $M2 --source-dest-check "{\"Value\": false}"
+    
+    aws ec2 create-tags --resources $M3 --tags Key=Name,Value=ciliumMaster3
+    aws ec2 create-tags --resources $M3 --tags Key=role,Value=master
+    aws ec2 modify-instance-attribute --instance-id $M3 --source-dest-check "{\"Value\": false}"
+
+::
+
+    core@host:~/cilium/installation$ 
+    aws ec2 create-tags --resources $S1 --tags Key=Name,Value=ciliumSlave1
+    aws ec2 create-tags --resources $S1 --tags Key=role,Value=slave
+    aws ec2 modify-instance-attribute --instance-id $S1 --source-dest-check "{\"Value\": false}"
+    
+    aws ec2 create-tags --resources $S2 --tags Key=Name,Value=ciliumSlave2
+    aws ec2 create-tags --resources $S2 --tags Key=role,Value=slave
+    aws ec2 modify-instance-attribute --instance-id $S2 --source-dest-check "{\"Value\": false}"
+    
+    aws ec2 create-tags --resources $S3 --tags Key=Name,Value=ciliumSlave3
+    aws ec2 create-tags --resources $S3 --tags Key=role,Value=slave
+    aws ec2 modify-instance-attribute --instance-id $S3 --source-dest-check "{\"Value\": false}"
+
+::
+
+    $ aws ec2 describe-instances --query 'Reservations[*].Instances[*].[InstanceId,PrivateIpAddress,PublicIpAddress,Tags[    0].Value,ImageId,State.Name,Placement.AvailabilityZone,LaunchTime]' --filters Name=tag:Name,Values=cilium* --output text |sort -k     4|grep -v None
+    
+    i-xxxxxxxxxxxxxxxxxxx   192.168.32.11   x.x.x.x ciliumMaster1   ami-xxxxxxxx    running us-west-1a  2017-06-15T13:02:34.000Z
+    i-xxxxxxxxxxxxxxxxxxx   192.168.32.12   x.x.x.x ciliumMaster2   ami-xxxxxxxx    running us-west-1a  2017-06-15T13:02:34.000Z
+    i-xxxxxxxxxxxxxxxxxxx   192.168.32.13   x.x.x.x ciliumMaster3   ami-xxxxxxxx    running us-west-1a  2017-06-15T13:02:34.000Z
+    i-xxxxxxxxxxxxxxxxxxx   192.168.32.21   x.x.x.x ciliumSlave1    ami-xxxxxxxx    running us-west-1a  2017-06-15T13:02:34.000Z
+    i-xxxxxxxxxxxxxxxxxxx   192.168.32.22   x.x.x.x ciliumSlave2    ami-xxxxxxxx    running us-west-1a  2017-06-15T13:02:34.000Z
+    i-xxxxxxxxxxxxxxxxxxx   192.168.32.23   x.x.x.x ciliumSlave3    ami-xxxxxxxx    running us-west-1a  2017-06-15T13:02:34.000Z
+
+
+
+2. Cluster TLS using OpenSSL
+----------------------------
+
+2.1 Kubernetes API Server Keypair
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+::
+
+    core@host:~/cilium$ cd installation/ca/
+    core@host:~/cilium/installation/caNew$ vi openssl.cnf
+    [req]
+    req_extensions = v3_req
+    distinguished_name = req_distinguished_name
+    [req_distinguished_name]
+    [ v3_req ]
+    basicConstraints = CA:FALSE
+    keyUsage = nonRepudiation, digitalSignature, keyEncipherment
+    subjectAltName = @alt_names
+    [alt_names]
+    DNS.1 = kubernetes
+    DNS.2 = kubernetes.default
+    DNS.3 = kubernetes.default.svc
+    DNS.4 = kubernetes.default.svc.cluster.local
+    IP.1 = 192.168.192.1
+    IP.2 = x.x.x.x
```

Comment:
Why is IP.2 the only one anonymized? Be consistent (all showing, or only an example and the rest anonymized).